Cohort1/tbd/server/controllers/recipe.js

```diff
@@ -1,18 +1,45 @@
 const express = require('express');
 const Recipe = require('../models/recipes');
+const { URLSearchParams } = require('url');
 
 const router = express.Router();
 
 //Index (GET all)
 router.get("/", async (req, res) => {
-    try {
-        const recipes = await Recipe.find({});
-        // console.log('search terms are: ' + req.query.q);
-        // console.log(typeof(req.query.q))
-        res.json(recipes);
-
-    } catch (err) {
-        res.json({ err });
+    // console.log(`path: ${req.protocol}://${req.hostname}${req.originalUrl}`);
```

Comment:
Nit: remove the console.log calls rather than commenting them out. We could also look to implement a logging framework that would allow us to put debug and info logs in place.